### PR TITLE
This is only a test of a single testcase

### PR DESCRIPTION
**Pull request by hamzy**
We are making sure the ppc64le CI is working

### CODE REVIEW

File: ci-operator/config/openshift/release/openshift-release-master__nightly-4.8.yaml [hunk 1]
```diff
@@ -471,8 +471,20 @@ tests:
     env:
       ARCH: ppc64le
       BRANCH: "4.8"
-      TEST_TYPE: conformance-parallel
-    workflow: openshift-e2e-libvirt
+    test:
+    - as: e2e-test
+      cli: latest
+      commands: openshift-tests run-test "[sig-api-machinery] Servers with support
```
Comment (deepsm007) [inline ci-operator/config/openshift/release/openshift-release-master__nightly-4.8.yaml:477]: you are missing --from-repository flag

Comment (deepsm007) [inline ci-operator/config/openshift/release/openshift-release-master__nightly-4.8.yaml:477]: Does this test require image from quay.io/multi-arch/community-e2e-images ?

**Review by hamzy — COMMENTED** [review]
good catch!

**Pull request closed without merging**